scene/2d/mesh_instance_2d.cpp

```diff
@@ -53,9 +53,27 @@ void MeshInstance2D::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::OBJECT, "texture", PROPERTY_HINT_RESOURCE_TYPE, "Texture2D"), "set_texture", "get_texture");
 }
 
+void MeshInstance2D::_mesh_changed() {
+	queue_redraw();
```

Comment:
If this method just calls `queue_redraw()`, then it's not really needed.

Reply:
Its possible there is something else that could be called but without it you still have the bug.   Manually calling this was the work around reported on the original issue.  It has always been called by set_mesh as well, it just now get called automatically if the mesh gets modified.  Likely some state gets updated during the process.  set_texture does similar.

Reply:
I mean you can remove `_mesh_changed` and call/connect to `queue_redraw()` directly.

Reply:
I'll investigate this some more though.  Bounding boxes looked sane when I added some debug draw.

Reply:
I removed _mesh_changed.